packages/frontend-2/components/settings/workspaces/security/sso/Wrapper.vue

```diff
@@ -126,6 +131,7 @@ graphql(`
         issuerUrl
       }
     }
+    hasAccessToSSO: hasAccessToFeature(featureName: OidcSso)
```

Comment:
As we do more of these we should turn these into fragments so that they always have the same prefix and thus rely on the same value in the cache. Currently there's only this and my "hasMultiRegionAccess" one afaik, but we should keep this in mind as we keep adding these fields.